test/test.py

```diff
@@ -19,6 +19,13 @@ def test_fmpz():
     assert flint.fmpz() == flint.fmpz(0)
     L = [0, 1, 2, 3, 2**31-1, 2**31, 2**63-1, 2**63, 2**64-1, 2**64]
     L += [-x for x in L]
+    for i in L:
+        f = flint.fmpz(i)
+        assert int(f) == i
+        assert flint.fmpz(f) == f
+        assert flint.fmpz(str(i)) == f
+    raises(lambda: flint.fmpz("qwe"), ValueError)
+    raises(lambda: flint.fmpz([]), TypeError)
```

Comment:
I just realised I'm using this `raises` function incorrectly because it works different from the `pytest/sympy` one. Is there any reason not to use pytest in general here in `python_flint`?

Reply:
No, I wouldn't mind.